.github/pull_request_template.md

```diff
@@ -2,4 +2,4 @@
 
 Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.
 
-Routes in this application are being migrated to another application, please check with #govuk-patterns-and-pages when making changes.
\ No newline at end of file
+**Routes in this application are being migrated to another application, please check with [#govuk-patterns-and-pages](https://gds.slack.com/archives/C06T3EFUUQ6) when making changes.**
```

Comment:
I wonder whether we should add something like 📣 to the beginning to make it stand out more, and/or have a heading like `Ongoing work to be aware of`,  and also move it to above the CD notice? I think at the moment it still blends in too much with the existing template.

Reply:
I agree, a heading would be good, and maybe a checklist item to confirm it's been considered? 

e.g.

- [ ] #govuk-patterns-and-pages have been notified of this change

Reply:
Done!